tekton/os/cp4i/12-ibmint-cp4i-build-task.yaml

```diff
@@ -13,19 +13,23 @@ metadata:
   name: cp4i-build
   namespace: cp4i
 spec:
-  # The security and environment settings are needed for OpenShift in a non-default
-  # namespace such as cp4i. Kaniko is expecting to be root in the container.
   stepTemplate:
-    securityContext:
-      runAsUser: 0
     env:
-      - name: "HOME"
-        value: "/tekton/home"
+      #- name: "HOME"
+      #  value: "/tekton/home"
```

Comment:
Commenting out /tekton/home won't cause issues?

Reply:
It seems to work fine because the credentials are in `/tekton/creds` so I can do
```
        - name: "DOCKER_CONFIG"
          value: "/tekton/creds/.docker/"
```
on the steps further down to pick them up.

Reply:
Yes, as long as it doesn't change in some future Tekton release! The minimal-image-build files still use the original HOME setting so the technique isn't lost.